test/k6/src/tests/instances.js

```diff
@@ -5,6 +5,7 @@
     -e env=*** `
     -e userId=*** `
     -e partyId=*** `
+    -e orgNumber=*** `
```

Comment:
I would add org number below existing user params. As to not create confusion of who the user and partyIds are linked to.